swipe.js

```diff
@@ -116,6 +116,8 @@
     }
 
     function slide(to, slideSpeed) {
+      slideSpeed = slideSpeed === undefined ? speed : slideSpeed;
```

Comment:
looks similar as code for default speed, this could be moved to a separate function, e.g:
```js
function getSlideSpeed(speed, defaultSpeed) {
  defaultSpeed = defaultSpeed || 300;
  return !isNaN(speed) ? Number(speed) : defaultSpeed;
}
```
and use it in both functions
```js
var speed = getSlideSpeed(options.speed);
```
```js
slideSpeed = getSlideSpeed(speed, slideSpeed);
```